cypress/e2e/3-learn/home.cy.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+describe('my awesome website', () => {
```

Comment:
Use the describe to describe the user story of this test set.
For example, file name could be `darkMode.cy.js`.
The describe would be:
`"As a user, i should be able to toggle light and dark mode themes"`